resources/views/dashboard/employee/index.blade.php

```diff
@@ -316,6 +292,23 @@ class: 'bi bi-plus-lg btn btn-outline-primary rounded-2 px-3 py-0' ,
                     ],
                 });
 
+                //Dropdown filter
+                $('.select2-filter').on('change', function () {
+                    const select = $(this)
+                    const ul = $(this).siblings('span.select2').find('ul')
+                    const count = $(this).select2('data').length
+                    if($(this).val().includes("clear-btn")) {
```

Comment:
remove this if block and use inbuild clear option

Reply:
implemented as mentioned please review again @Raza9798 #96